alpha/apps/kaltura/lib/access_control/model/actions/kAccessControlAddRequestHostPrefixAction.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model.data
+ */
+class kAccessControlAddRequestHostPrefixAction extends kRuleAction 
```

Comment:
What if we will want in the future a suffix or infix? I would name it differently and support any URL modification.